docs/src/shutdown.md

```diff
@@ -0,0 +1,16 @@
+# Shutdown
+
+It is not possible to explicitly close connection to the database.
+Given connection is closed, when the client varriable assosiated with this connection,
+is collected by the garbage collector.
```

Comment:
:wrench:

Reply:
```suggestion
A connection is closed when the client variable associated with this connection is collected by the garbage collector.
```

Reply:
Please be aware of the distinction that defining vs non-defining clauses present.

https://diposit.ub.edu/dspace/bitstream/2445/180172/9/U3_Relative%20clauses_JITT.pdf